conda_build/meta.yaml

```diff
@@ -1,8 +1,11 @@
 about:
   home: https://gitub.com/ESGF/output_viewer
+build:
+  script: python setup.py install
+  noarch: python
 package:
   name: output_viewer
-  version: 1.2.5
+  version: 1.2.6
 requirements:
   build:
   - python {{ python }}
```

Comment:
For a `noarch: python` package, you shouldn't have the `{{ python }}` jinja template because the package comes out with python pinned to whichever version was used to build the package.  This was the problem with the conda-forge build as well.

Reply:
It's not critical in `build`, so maybe not an issue but it definitely matters in `run`.